src/platform/forms/save-in-progress/SaveInProgressErrorPage.jsx

```diff
@@ -66,13 +66,14 @@ class SaveInProgressErrorPage extends React.Component {
 
   render() {
     const { loadedStatus, formConfig } = this.props;
-    const { forbidden, noAuth, notFound } = formConfig.savedFormMessages || {};
```

Comment:
I think  we should've kept it this way.

Reply:
Also, these were properties of `formConfig.savedFormMessages`, not directly `formConfig`. Did that change?

Reply:
No, this was a mistake. I added that back.